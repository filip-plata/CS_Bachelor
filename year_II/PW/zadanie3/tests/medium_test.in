8 4 5
1 x[2] = (x[3] * (-x[4]))
2 x[3] = ((x[1] + x[1]) + (x[1] + x[1]))
3 x[0] = (x[2] * (x[4] + 7))
4 x[1] = (-7)
5 x[2] 1 x[4] 3
6 x[1] 666
7 x[1] 1 x[4] 1
8 x[0] 1032
