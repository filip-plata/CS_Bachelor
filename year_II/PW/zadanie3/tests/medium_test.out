1 P
2 P
3 P
4 P
5 P 10
6 F
7 P -32
8 P 1032
